content/en/docs/languages/net/instrumentation.md

```diff
@@ -281,6 +281,13 @@ var activity = Activity.Current;
 Note that `using` is not used in the prior example. Doing so will end current
 `Activity`, which is not likely to be desired.
 
+Alternatively you can use the `Tracer` class to obtain the current `TelemetrySpan` to stay within the OpenTelemetry namespace.
```

Comment:
if we are documenting this approach here, then it should be shown throughout the doc. Else it'll be inconsistent.

Reply:
I think having an alternative approach here in the doc makes things confusing to the user. @Philly-B would you elaborate what problem do you want to solve here? Thanks!

Reply:
Since this is already documented in the shim doc I feel like it's fine not to document here: https://opentelemetry.io/docs/languages/net/shim/#get-the-current-span

Reply:
I think you are right, that is indeed a bad spot to add that information.
I think my confusion was based on the first examples in the documentation being in the System.Diagnostics namespace and I only later found the shim documentation. 
Thank you for your reply, I think you are right and that documentation hint would not add anything but would rather make it more complicated.